.github/workflows/faux-cdm-ci.yaml

```diff
@@ -0,0 +1,17 @@
+name: faux-cdm-ci
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+    - LICENSE
+    - '**.md'
+    - .github/workflows/faux-crucible-ci.yaml
+    - .github/workflows/faux-cdm-ci.yaml
+    - 'docs/**'
+
+jobs:
+  cdm-ci:
```

Comment:
this job name needs to match the job name in the non-faux case so that either one can meet the requirement for PR merging (depending on the path requirements)